src/api/routes/stats.ts

```diff
@@ -0,0 +1,46 @@
+import { TypeBoxTypeProvider } from '@fastify/type-provider-typebox';
+import { FastifyPluginAsync, FastifyPluginCallback } from 'fastify';
+import { Server } from 'http';
+import { blockParam } from '../util/helpers';
+import { Type } from '@sinclair/typebox';
+import { BlockHeightParam } from '../schemas';
+
+const IndexRoutes: FastifyPluginCallback<Record<never, never>, Server, TypeBoxTypeProvider> = (
+  fastify,
+  options,
+  done
+) => {
+  fastify.get(
+    '/stats/inscriptions',
+    {
+      schema: {
+        operationId: 'getInscriptionCount',
+        summary: 'Inscription Count',
+        description: 'Retrieves count of inscriptions revealed per block',
+        tags: ['Inscriptions'],
```

Comment:
Use a new tag here, something like `Statistics`. You'll also need to list it in the OpenAPI config object (I think that's in the init.ts for the API)